api/Document.json

```diff
@@ -246,11 +246,11 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/alinkColor",
           "support": {
             "chrome": {
-              "version_added": "64",
+              "version_added": "1",
```

Comment:
Can we figure this one out as part of https://github.com/mdn/browser-compat-data/issues/6839, along with anything that has notes for `HTMLDocument`?

Since we don't have an api.HTMLDocument.alinkColor entry, I think the best we can do here is to have two entries, the first being `partial_implementation`. The difference is not entirely academic, since `document.alinkColor` won't work in SVG and XML documents before the change.

Reply:
I've separated these changes out as #6865!